lib/domain/entities/volunteer.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:nowingoogle/domain/entities/user.dart';
+import 'package:nowingoogle/domain/enums/user_roles.dart';
+import 'package:nowingoogle/presentation/pages/constants/color_scheme.dart';
+
+class Volunteer extends User {
+  final bool isPOC;
```

Comment:
What does isPOC means?

Reply:
isPOC is used to identify if the volunteer is a lead for his domain and will the responsibility of coordinating with his team and planning tasks, before taking it to the next hierarchy, i.e, organizers.